### DISCUSSION TITLE
[Feature Request]

### DISCUSSION THREAD

**Discussion by Antvictor** [vuepress-theme-hope/vuepress-theme-hope#2990] [Q&A] [answered]
### Checklist

- [X] I searched existing issues and no one else requests similar feature.
- [X] I think that 25%+ users are positive towards this feature.

### Describe the feature

大佬您好，我在使用选项卡时，发现在@tab之后面写的文字会被统一放在一个字符串，我在使用时想给一部分字段加一些颜色。
一个例子：
```
::: tabs
@tab hello <span style="color:red">推荐</span>
hello
@tab world
world 
:::
```
然后解析后页面把span标签也输出了。
想请问一下，有没有什么可以让一部分字段加颜色呢？或者说可以将标签当作有效的html输出呢

**Comment by Mister-Hope** [comment, marked as answer]
自己写一个新插件，使得title部分支持HTML。
